textdb/textdb-api/src/main/java/edu/uci/ics/textdb/api/common/IDictionary.java

```diff
@@ -0,0 +1,12 @@
+/**
+ * 
+ */
+package edu.uci.ics.textdb.api.common;
+
+/**
+ * @author Sudeep [inkudo]
+ *
+ */
+public interface IDictionary {
+    String getNextDictValue();
```

Comment:
A minor optional comment: change "getNextDictValue" to "getNextValue" since "Dict" is really not needed.